### PR TITLE
Added a 'No matches found' div for Module Manager::Login::Menu Assignment.

### PR DESCRIPTION
**Pull request by prasoon2211**
A div added to the end of the list that is displayed when the result is empty.

screenshot from 2015-01-16 23 26 52

### CODE REVIEW

**Comment by prasoon2211** [comment]
Can someone please look at this? This fix is pretty minor. Let me know if I should change anything.

File: media/jui/js/treeselectmenu.jquery.js [hunk 1]
```diff
@@ -51,15 +51,23 @@ jQuery(function($)
...
 		{
 			if ($(this).text().toLowerCase().indexOf(text) == -1) {
 				$(this).hide();
+				hidden++;
 			}
 			else {
 				$(this).show();
 			}
 		});
+		if(hidden == $list_elements.length)
```
Comment (dgt41) [inline media/jui/js/treeselectmenu.jquery.js]: can you just replace this with `if($list_elements.length == 0)` and also remove the `var hidden = 0;` few lines above?

Comment (prasoon2211) [inline media/jui/js/treeselectmenu.jquery.js]: No problem :) Anything else?

File: administrator/components/com_modules/views/module/tmpl/edit_assignment.php [hunk 2]
```diff
@@ -122,6 +122,7 @@ function menuHide(val)
 					<?php endif; ?>
 				<?php endforeach; ?>
 			</ul>
+			<div id="noresultsfound" style="display:none;"><?php echo JText::_('COM_MODULES_NO_ITEMS'); ?></div>
```
Comment (zero-24) [inline administrator/components/com_modules/views/module/tmpl/edit_assignment.php]: @prasoon2211 can you move the php code in a own line like:

```
<div id="noresultsfound" style="display:none;">
<?php echo JText::_('COM_MODULES_NO_ITEMS'); ?>
</div>
```

(please also add a tab bevor the PHP code. i can't show it here on github code view :smile:

**Comment by zero-24** [comment]
@prasoon2211 any how (don't ask me why) I'm unable to test this here successful. Other words the message will not show to me.

I tryed both com_patchtester and a clean install of your branche but both dont work.

**Comment by brianteeman** [comment]
@test tried and no difference<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5766">issues.joomla.org/joomla-cms/5766</a>.</sub>

**Comment by dgt41** [comment]
@test OK if debug is on
Which means that the file `media/jui/js/treeselectmenu.jquery.*min*.js` needs to be *updated* as well
screen shot 2015-01-18 at 10 30 03

@prasoon2211 can you minify the js?

**Comment by prasoon2211** [comment]
Ok, I've minified the JS and added the changes that @zero-24 suggested.

**Comment by infograf768** [comment]
Please, do not create a new language string when we have a global string:
JGLOBAL_NO_MATCHING_RESULTS="No Matching Results"

also, should not we use the same formatting as elsewhere in admin i.e.
```
			<div id="noresultsfound" style="display:none;" class="alert alert-no-items">
				<?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
			</div>
```
which gives:

screen shot 2015-01-19 at 08 53 06

**Comment by prasoon2211** [comment]
@infograf768 Okay. It looks like everything is done. Should I squash the commits?

**Comment by brianteeman** [comment]
@test works for me now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5766">issues.joomla.org/joomla-cms/5766</a>.</sub>

**Comment by zero-24** [comment]
Thanks @test successful moving to RTC.

>Should I squash the commits?

Yes Thanks @prasoon2211 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5766">issues.joomla.org/joomla-cms/5766</a>.</sub>

**Pull request closed without merging**